src/views/questions-list.js

```diff
@@ -1,17 +1,54 @@
 import { quizData } from "../data.js";
+import { selectAnswer } from "../handlers/select-answer.js";
+/* import { selectAnswer} from '../handlers/select-answer.js'; */
 
 export function createQuestionList() {
-  const list = document.createElement('ul');
+//    const list = document.createElement('ul');
   
-    quizData.questions.forEach(item =>{
-        const questionsLi = document.createElement('li')
-        list.appendChild(questionsLi)
-        questionsLi.textContent = item.text
-    // console.log(item.text)
-})
+//      quizData.questions.forEach(item =>{
+//          const questionsLi = document.createElement('li')
+//          list.appendChild(questionsLi)
+//          questionsLi.textContent = item.text
+//      // console.log(item.text)
+//  })
   
     
   
-  // TODO: actually put things into the list
-  return list;
+// //   // TODO: actually put things into the list
+//    return list;
 }
+
+
+export function getRandomQuestion(item){
+  const randomQuestion = Math.floor(Math.random(item) * item.length) 
+  return item[randomQuestion]
+};
+
+
+
+export function displayQuestion(){
+  const question = getRandomQuestion(quizData.questions);
+  const answers = question.answers;
+
+  const questionArea = document.getElementById('question');
+  questionArea.innerHTML = question.text;
+  
+  Object.entries(answers).forEach(answer => {
```

Comment:
Up to you, but I think `for (const [answerKey, answer] of Object.entries(answers)) {` is much nicer. Even without the fancy array destructuring `for (const answerEntry of Object.entries(answers)) {` this would be nicer to read :)